pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_import_modules.adoc

```diff
@@ -0,0 +1,19 @@
+[id="import_modules_{context}"]
+
+= Importing modules into {ProductShortName}
+
+Importing a module into {ProductShortName}.
+
+.Prerequisites
+
+* You are assigned a publisher or admin role in {ProductShortName}.
```

Comment:
I would say 'publisher or author' here. Currently these are the only 2 roles in the system, so practically speaking, that reduces the prerequisite to "you're logged in."

Although our environments do have an admin login currently, using the system as the admin user is unsupported. This would be particularly problematic once we implement some form of access control (i.e., you can only see the repositories you're associated with). It would be impossible to prevent the admin user from seeing everything regardless.

We may add more roles in the future (perhaps an admin role, distinct from the admin user), but those details haven't been ironed out.